packages/modal/src/Modal/Modal.tsx

```diff
@@ -27,6 +27,7 @@ import ModalView from './ModalView';
  */
 const Modal = React.forwardRef((props: ModalProps, ref: ForwardedRef) => {
   return (
+    /** TODO @steph: remove `PopoverProvider` once Modal uses `useOverlay` */
```

Comment:
Nit: If there's a ticket where this is getting done, could be helpful to reference it here.